canonical-web-and-design/snapcraft.io
# Add progressive release column to history table

## Done
Added the progressive releases column with chart to the releases history table

## QA
- Go to https://snapcraft-io-3964.demos.haus/notion-snap/releases
- Expand the progressive release table row
- Check that there is a "Release progress" table column
- Check that there is a chart that accurately represents the progressive release

## Issue
Fixes #3947, https://github.com/canonical-web-and-design/marketplace-tribe/issues/2333
